### PR TITLE
Add marcuscalidus-svg-panel 0.1.0

### PR DESCRIPTION
**Pull request by MarcusCalidus**
The Panels ist now ready for IE11 & Grafana 5
as described here: https://community.grafana.com/t/panel-with-svg-and-overlayed-values-for-iiot/2953/12

### CODE REVIEW

**Comment by MarcusCalidus** [comment]
panel was renamed in accordance with docs.grafana.org/plugins/developing/code-styleguide/

**Comment by daniellee** [comment]
Just took a quick look today and noticed one easy thing that needs to be fixed. The screenshot images in the README.md need to be links to GitHub rather than relative links. The readme is shown both in the plugins section in Grafana and on Grafana.com. This is the easiest way to format the url: 

https://raw.githubusercontent.com/MarcusCalidus/marcuscalidus-svg-panel/master/dist/img/inAction.png

**Comment by MarcusCalidus** [comment]
Hi daniellee,
thank you for your input. URLs have been changed.

**Comment by MarcusCalidus** [comment]
Hi Danielle,
is there any input on this plugin? If there's still something missing just drop me a line please.

cheers
Marco

**Comment by Chris-Hicks** [comment]
+1

**Comment by daniellee** [comment]
Hi @MarcusCalidus sorry for the long delay - the main Grafana project and other stuff is taking a lot of our time these days.

I reviewed this plugin and there is only one thing left to do before publishing:

All image links in the README.md have to be absolute, Grafana cannot convert relative links into absolute links like GitHub can:

image

### Feedback

- The metadata looks fine and the readme looks good and it explains the concepts.
- It feels like you have created a mini IDE for writing your own panel plugin in Grafana which is pretty cool. It will be interesting to see how many downloads it gets.
- I have on my docs backlog to document the MetricsPanelCtrl class - as it is, might be a bit hard for new users to figure out how to use it in onInit and onHandleMetric.

If you can fix the links in the readme then I will publish it straight away.

**Comment by MarcusCalidus** [comment]
HI daniellee,

thank you for your feedback. The intention was to have a kind of IDE like plugin. There should be other great things like code completion etc. but I am also short of resources for that. 
A documentation for the MetricsPanelCtrl would be great. I could link to that ;-).

As for the changes I thought the commit 2447f528a168e6ae4e00eeecb031732a0e133865 already handles that. It's the commit I refer to in the repository json. I have absolute paths to https://raw.githubusercontent.com/ I wonder why it is not displayed in your browser. Is there any way I can test the repo.json in my own Grafana installation?

image

**Comment by daniellee** [comment]
@MarcusCalidus might be my fault. Will test again.

**Comment by daniellee** [comment]
Ok, took me a while to figure this out. You have another copy of the readme here: 

https://raw.githubusercontent.com/MarcusCalidus/marcuscalidus-svg-panel/master/dist/README.md

Grafana first checks the dist folder for a README.md if it does not exist then it uses the README.md from the root.

You can see which version of the README.md is loaded by going to the plugins section in Grafana:

http://localhost:3000/plugins/marcuscalidus-svg-panel/edit

**Comment by MarcusCalidus** [comment]
thx daniel.
I now committed with correct (hopefully) /dist folder. It should be good by now.
let's see how that'll work out.
Have a great weekend.

Marco

File: repo.json [hunk 1]
```diff
@@ -2041,6 +2041,11 @@
           "version": "0.1.0",
```
Comment (y2sarakawa) [inline repo.json]: Is it necessary to keep the version 0.1.0 there?

Comment (y2sarakawa) [reply repo.json]: Current commit points to the version before the dist directory was updated.
It is probably better that the version 0.1.0 refers the commit of f6dd083c0c6a241f6a6b82f4f01cbf702ddeaf72.

Comment (MarcusCalidus) [reply repo.json]: You are right 0.1.0 is not really necessary since there are no breaking changes. 
Fixed with the PR, for which I am really grateful.

**Comment by daniellee** [comment]
Finally merged and published: https://grafana.com/plugins/marcuscalidus-svg-panel

**Pull request merged**